source/fundamentals/builders/projections.txt

```diff
@@ -11,3 +11,320 @@ Projections Builders
    :class: singlecol
 
 .. _projections-builders:
+
+Overview
+--------
+
+MongoDB supports **field projection**, specifying which fields to include and exclude when returning results from a
+query. Projection in MongoDB follows some basic rules:
+
+- The ``_id`` field is **always** included and must explicitly be excluded
+- Specifying a field for inclusion implicitly excludes all other fields **except** the ``_id`` field
+- Specifying a field for exclusion removes only that field in a query result
+
+Find more information about projection mechanics :manual:`here </tutorial/project-fields-from-query-results/>`.
+
+The :java-core-api:`Projections <com/mongodb/client/model/Projections.html>` class provides static factory methods for
+all the MongoDB projection operators. Each method returns an instance of the :ref:`Bson <bson>` type which you can pass
+to any method that expects a projection.
+
+.. tip::
+
+   For brevity, you may choose to import the methods of the ``Projections`` class statically:
+
+   .. literalinclude:: /includes/fundamentals/code-snippets/builders/Projections.java
+      :start-after: begin static import
+      :end-before: end static import
+      :language: java
+      :dedent:
+
+   The examples below assume this static import.
+
+The examples also use the following setup in addition to the ``Projections`` static import:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/Projections.java
+   :start-after: begin imports
+   :end-before: end imports
+   :language: java
+   :dedent:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/Projections.java
+   :start-after: begin declarations
+   :end-before: end declarations
+   :language: java
+   :dedent:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/Projections.java
+   :start-after: begin collection setup
+   :end-before: end collection setup
+   :language: java
+   :dedent:
+
+Inclusion
+---------
+
+Use the ``include()`` method to specify the inclusion of one or more fields.
+
+The following example includes the ``year`` field and (implicitly) the ``_id`` field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/Projections.java
+   :start-after: begin include single field
+   :end-before: end include single field
+   :language: java
+   :dedent:
+
+The output should resemble:
```

Comment:
This may be 100% personal preference, but I always think the words preceding the colon should be a complete sentence, like: The following code shows the output from this projection:

¯\\_(ツ)_/¯

Reply:
(As you already do preceding each main code snippet)